### PR TITLE
Fixed a formatting error

### PR DESCRIPTION
**Pull request by hessio**
Sign-off-by: Patrick Hession <patrick.hession@seagate.com>

### Changes
 - Removed extra spaces that prevented an easy copy/paste for the user

-----
[View rendered doc/ProvisionReleaseBuild.md](https://github.com/Seagate/cortx/blob/hessio-patch-2/doc/ProvisionReleaseBuild.md)

### CODE REVIEW

File: doc/ProvisionReleaseBuild.md [hunk 1]
```diff
@@ -13,30 +13,30 @@
...
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
-      yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/cortx_iso/"
-
-      cat <<EOF >/etc/pip.conf
-      [global]
-      timeout: 60
-      index-url: $CORTX_RELEASE_REPO/python_deps/
-      trusted-host: $(echo $CORTX_RELEASE_REPO | awk -F '/' '{print $3}')
-      EOF
+   yum-config-manager --add-repo "${CORTX_RELEASE_REPO}/3rd_party/"
```
Comment (huanghua78) [inline doc/ProvisionReleaseBuild.md:16]: Apparently, this is wrong.

This change is not needed.
But, we need a fix for the Line 15.
"yum install -y yum-utils" should be on a new line.

**Review by huanghua78 — CHANGES_REQUESTED** [review]
This fix is wrong.

**Comment by huanghua78** [comment]
Also, please fork and clone the "cortx" repo, do some changes, and create new PR.
Not on the Seagate repo.

**Comment by hessio** [comment]
Duplicate

**Pull request closed without merging**